website/docs/guides/markdown-features/markdown-features-inline-toc.mdx

```diff
@@ -53,12 +47,19 @@ import TOCInline from '@theme/TOCInline';
 <TOCInline
   // Only show h2 and h4 headings
   toc={toc.filter((node) => node.level === 2 || node.level === 4)}
+  minHeadingLevel={2}
```

Comment:
seems weird to use filter + min/max at the same time?

Reply:
`filter` filters out the h3 nodes. But when the h4 nodes are passed into `TOCInline`, they are still being ignored by the component if `maxHeadingLevel` is not configured to be 4`. I don't think there's much we can do, because no-one knows that a `filter` happened on the user-side.

Reply:
I see thanks